### PR TITLE
Create delete user by admin

### PR DESCRIPTION
**Pull request by samnangcattor**

### CODE REVIEW

File: app/controllers/admins/sessions_controller.rb [hunk 1]
```diff
@@ -6,7 +6,10 @@ def new
   def create
     user = User.find_by email: params[:admin][:email]
     unless user && user.admin?
```
Comment (RathanakSreang) [inline app/controllers/admins/sessions_controller.rb]: Use before action for checking user.

File: app/controllers/admins/sessions_controller.rb [hunk 2]
```diff
@@ -1,12 +1,20 @@
-class Admins:SessionsController < Desive::SessionsController
+class Admins::SessionsController < Devise::SessionsController
+  before_action :set_check_user
```
Comment (RathanakSreang) [inline app/controllers/admins/sessions_controller.rb]: only create

File: app/controllers/admins/sessions_controller.rb [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
   def new
     super
   end
 
   def create
+    super
+  end
+
+  private
+  def set_check_user
```
Comment (HorNorin) [inline app/controllers/admins/sessions_controller.rb]: can you change this method name to something better?

**Comment by RathanakSreang** [comment]
Ok

**Comment by HorNorin** [comment]
ok

**Pull request merged**